test/e2e/specs/editor/blocks/navigation-colors.spec.js

```diff
@@ -271,6 +271,63 @@ test.describe( 'Navigation colors', () => {
 		await colorControl.testFrontendColors( expectedNavigationColors );
 	} );
 
+	test( 'The submenu/overlay colors do not apply to all navigation links unless the overlay is open', async ( {
```

Comment:
these changes seem unrelated? (thru line 330)

Reply:
Yes these are not directly related. This adds coverage for something [I broke while trying to clean up some styles](https://github.com/WordPress/gutenberg/pull/69720/commits/f550145e201402ccc95704372c6947ac60e29346#r2021845945). It’s possible that one of the existing tests could be modified so it would fail instead of adding this new one but I figured this is more explicit.